conanfile.py

```diff
@@ -8,7 +8,7 @@
 
 class NuRaftMesgConan(ConanFile):
     name = "nuraft_mesg"
-    version = "2.2.2"
+    version = "2.3.2"
```

Comment:
Not critical, but 2.3.1. 
I can go into why not 2.3.0, has to do with ranged versions and the include_prerelease we use when depending on this though.